### ISSUE TITLE
Typescript typings not found

### ISSUE THREAD

**Issue by aecz** [rstacruz/pnpm#404]
pnpm v0.39.0
node v6.5.0
OS Windows 7
### Code to reproduce the issue:

```
mkdir typings-test
cd typings-test
pnpm install typescript@1.8.10 mocha @types/mocha
mkdir test
cd ..
```

Create a `test.ts` file with the following content:

```
describe('Test', () => {    
    describe('run', () => {
        it('should run', () => {
            console.log('i run');
        });
    });
});
```

Then run:

```
node_modules/.bin/tsc test/test.ts
node_modules/.bin/mocha
```
### Expected behavior:
- test.ts get compiled into test.js
- mocha runs and passed
### Actual behavior:
- test.ts fails compilation with error 'Cannot find name describe'
- mocha does not find any test files (look for *.js)

I suggest we could copy the @types packages into node_modules instead of linking them. Seems to solve the issue as commented here: https://github.com/rstacruz/pnpm/issues/394#issuecomment-251905995

**Comment by zkochan** [comment]
I really hope this will be fixed on the TypeScript side... It would look really weird to have such hacks in the pnpm codebase

**Comment by zkochan** [comment]
Maybe we can fall back to npm for some "blacklist" of popular packages...

**Comment by aecz** [comment]
What about a flag --copy-typings or --no-link-typings ? Does not affect pnpm users and if Typescript fixes it then no need to set flag :)

**Comment by zkochan** [comment]
I understand, I am talking about the implementation. Seems like it will be the same that npm does, so we can just run `npm install` inside pnpm for those packages

**Comment by zkochan** [comment]
Maybe hard links would solve this issue.

**Comment by zkochan** [comment]
This is going to be fixed in TypeScript 2.1.0. If you use typescript@next, it will work already

**Comment by zkochan** [comment]
I am closing this because the latest TypeScript is 2.1.0 now and it has all the symlink issues solved. So this will work with the latest TypeScript
